src/css/controls/imports/slider.less

```diff
@@ -190,6 +190,38 @@
     }
 }
 
+.jw-flag-audio-player,
+.jwplayer:not(.jw-breakpoint--1):not(.jw-breakpoint-0):not(.jw-breakpoint-1) {
+    .jw-horizontal-volume-container {
+        transition: width 300ms @default-timing-function;
+        width: 0;
+
+        &.jw-open {
+            width: @volume-rail-length;
+
+            .jw-slider-volume {
+                padding-right: 24px;
+                transition: opacity 300ms;
+                opacity: 1;
+            }
+        }
+
+        .jw-slider-volume {
+            opacity: 0;
+
+            .jw-knob {
+                transform: translate(-50%, -50%);
+            }
+        }
+    }
+
+    .jw-button-container {
+        .jw-icon {
+            flex: 0 0 auto;
```

Comment:
This styling was added because at lower breakpoints (2 & 3 depending on how many items are in the controlbar), when the volume slider opens it would cause the surrounding elements to squash and shrink to make space. This was resulting in jumpy behavior since it would expand in both directions, so I added this to prevent the elements from doing that. 
Open to feedback and ideas for this styling.